code/modules/holiday/holidays.dm

```diff
@@ -661,3 +661,94 @@
+
+/datum/holiday/remembrance_day
+	name = "Remembrance Day"
+	begin_month = NOVEMBER
+	begin_day = 11
+	drone_hat = /obj/item/food/grown/poppy
+
+/datum/holiday/remembrance_day/getStationPrefix()
+	return pick("Peace", "Armistice", "Poppy")
+
+/datum/holiday/un_day
+	name = "Anniversary of the Foundation of the United Nations"
+	begin_month = OCTOBER
+	begin_day = 24
+
+/datum/holiday/un_day/greet()
+	return "On this day in 1945, the United Nations was founded, laying the foundation for humanity's united government!"
+
+/datum/holiday/un_day/getStationPrefix()
+	return pick("United", "Cooperation", "Humanitarian")
+
+//Gary Gygax's birthday, a fitting day for Wizard's Day
+/datum/holiday/wizards_day
+	name = "Wizard's Day"
+	begin_month = JULY
+	begin_day = 27
+	drone_hat = /obj/item/clothing/head/wizard
+
+/datum/holiday/wizards_day/getStationPrefix()
+	return pick("Dungeon", "Elf", "Magic", "D20", "Edition")
+
+//Species cultural holidays, based on days of significant PRs relating to their species
+//Tiziran Unification Day is celebrated on Sept 1st, the day on which lizards were made a roundstart race
+/datum/holiday/tiziran_unification
+	name = "Tiziran Unification Day"
+	begin_month = SEPTEMBER
+	begin_day = 1
+
+/datum/holiday/tiziran_unification/greet()
+	return "On this day over 400 years ago, Lizardkind first united under a single banner, ready to face the stars as one unified people."
+
+/datum/holiday/tiziran_unification/getStationPrefix()
+	return pick("Tizira", "Lizard", "Imperial")
+
+//The Festival of Atrakor's Might (Tizira's Moon) is celebrated on June 15th, the date on which the lizard visual revamp was merged (#9808)
+/datum/holiday/atrakor_festival
+	name = "Festival of Atrakor's Might"
+	begin_month = JUNE
+	begin_day = 15
+
+/datum/holiday/atrakor_festival/greet()
+	return "On this day, the Lizards traditionally celebrate the Festival of Atrakor's Might, where they honour the moon god with lavishly adorned clothing, large portions of food, and a massive celebration into the night."
+
+/datum/holiday/atrakor_festival/getStationPrefix()
+	return pick("Moon", "Night Sky", "Celebration")
+
+//Draconic Day is celebrated on May 3rd, the date on which the Draconic language was merged (#26780)
+/datum/holiday/draconic_day
+	name = "Draconic Language Day"
+	begin_month = MAY
+	begin_day = 3
+
+/datum/holiday/draconic_day/greet()
+	return "On this day, Lizardkind celebrates their language with literature and other cultural works."
+
+/datum/holiday/draconic_day/getStationPrefix()
+	return pick("Draconic", "Literature", "Reading")
+
+//Fleet Day is celebrated on Jan 19th, the date on which moths were merged (#34498)
+/datum/holiday/fleet_day
+	name = "Fleet Day"
+	begin_month = JANUARY
+	begin_day = 19
+
+/datum/holiday/fleet_day/greet()
+	return "This day commemorates another year of successful survival aboard the Mothic Grand Nomad Fleet. Moths galaxywide are encouraged to eat, drink, and be merry."
```

Comment:
drink what?

Reply:
never heard the phrase "to eat, drink, and be merry" before?

Reply:
what do they drink EOB, what do they drink

Reply:
moth milk

Reply:
I guess I'll need to wrap up the lizard food PR and move onto moths to answer that question, huh?